api/apps/api/test/steps/given-user-is-created.ts

```diff
@@ -0,0 +1,35 @@
+import { AccessToken } from '@marxan-api/modules/authentication/authentication.service';
+import { User } from '@marxan-api/modules/users/user.api.entity';
+import { INestApplication } from '@nestjs/common';
+import { getRepositoryToken } from '@nestjs/typeorm';
+import { hash } from 'bcrypt';
+import * as faker from 'faker';
+import * as request from 'supertest';
+import { Repository } from 'typeorm';
+
+export const GivenUserIsCreated = async (
```

Comment:
👏🏼 I sense this is going to be the groundwork for long overdue and welcome changes to how we set up users for tests 😉